Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM openshift/origin-release:golang-1.10 as builder
 
 WORKDIR /go/src/github.com/aerogear/mobile-security-service
 COPY . .
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
In order to keep our Docker image as small as possible, we should combine these three into a single `RUN` command. Each command in a Dockerfile adds a new layer to the image.

```dockerfile
RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh \
&& dep ensure \
&& go build -o mobile-security-service ./cmd/mobile-security-service/main.go
```